### PR TITLE
Public Dashboards: Docs: add ENV method to enable

### PR DESCRIPTION
**Pull request by bobemoe**
**What is this feature?**

Documentation on how to enable public dashboards with ENV variable (for docker)

**Why do we need this feature?**

It took me a while figuring out how to enable public dashboards in Docker. Many references online show
`GF_FEATURE_TOGGLES_PUBLICDASHBOARDS=enabled` or `GF_FEATURE_TOGGLES_PUBLICDASHBOARDS=true` neither of which works for me. Finally I found a working method of `GF_FEATURE_TOGGLES_ENABLE=publicDashboards` 

Since this seems to be missing from the official page, and the erroneous methods still seem to come first in search results I thought I would add it to the official docs.

**Who is this feature for?**

Docker users

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/dashboards/dashboard-public/index.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Add the `publicDashboards` feature toggle to your `custom.ini` file.
 publicDashboards = true
 ```
 
+Or if configuring using (Docker) enviroment variable:
```
Comment (chri2547) [inline docs/sources/dashboards/dashboard-public/index.md:35]:
```suggestion
If you are using Docker, you can enable public dashboards using an environment variable:
```

**Review by chri2547 — COMMENTED** [review]
A suggestion.

**Comment by chri2547** [comment]
@owensmallwood, @evictorero , could one of you please provide a tech review of this PR. Thank you.

**Review by evictorero — APPROVED** [review]
Thanks for adding this to the docs. Please add @chri2547 suggestion but LGTM.

**Review by chri2547 — APPROVED** [review]

**Comment by bobemoe** [comment]
Still relevant I think?

**Pull request merged**